code/controllers/AssetAdmin.php

```diff
@@ -25,28 +16,6 @@ class AssetAdmin extends LeftAndMain implements PermissionProvider{
 	private static $page_length = 15;
 	
 	/**
-	 * @config
-	 * @see Upload->allowedMaxFileSize
-	 * @var int
-	 */
-	private static $allowed_max_file_size;
-	
-	private static $allowed_actions = array(
-		'addfolder',
-		'delete',
-		'AddForm',
-		'DeleteItemsForm',
-		'SearchForm',
-		'getsubtree',
-		'movemarked',
-		'removefile',
-		'savefile',
-		'deleteUnusedThumbnails' => 'ADMIN',
-		'doSync',
-		'filter',
-	);
```

Comment:
So by removing the whitelist, we allow all public methods? And `deleteUnusedThumbnails` becomes available to everybody with access to the controller?